packages/docs/docs/getting-started/create-unirep-app.md

```diff
@@ -277,20 +275,23 @@ unirep.attesterSignUp(epochLength);
 See [attesterSignUp](../contracts-api/unirep-sol.md#attestersignup)
 :::
 
-### Change attestation policy
+### Customize attestation policy
 
-If users can request as much data as they want, the data will not be valuable in this app. 🙁 <br/>
-Try to give each epoch key only `1` data 0 value in each epoch.
+Attesters define their own systems to format user data and regulate how that data is attributed to users. An application's user data might represent aspects of a user's status, actions, associations, reputation, etc.<br/>
+The attester may implement its policy with any combination of smart contract code (customizing `UnirepApp.sol`) and traditional application logic. (The application may require a user to satisfy an on-chain OR an off-chain condition in order to receive new data.)
 
-Here is an example of creating a simple mapping to track if an epoch key recieved a value after an attetation was submitted to that epoch key:
+<!-- If users can request as much data as they want, the data will not be valuable in this app. 🙁 <br/>
+Try to give each epoch key only `1` data 0 value in each epoch. -->
+**e.g.**<br/>
+An attester wants to limit the amount of data a user can receive in each epoch. The code below utilizes a simple mapping to prevent an attestation to an epoch key if it has already recieved a value:
 
 ```sol
 mapping(uint => bool) epochKeyReceivedValue;
 
```

Comment:
@vivianjeng please check if the rewording here is accurately expressing your original idea

Reply:
yes correct!